dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/config/DLCOracleAppConfig.scala

```diff
@@ -104,13 +104,7 @@ case class DLCOracleAppConfig(
     config.getConfig("bitcoin-s.server")
   }
 
-  def rpcPortOpt: Option[Int] = {
-    if (serverConf.hasPath("bitcoin-s.server.rpcport")) {
-      Some(serverConf.getInt("bitcoin-s.server.rpcport"))
-    } else {
-      None
-    }
-  }
+  def rpcPort: Int = config.getInt("bitcoin-s.server.rpcport")
```

Comment:
I think this needs to be `bitcoin-s.oracle.rpcport` i think?

Reply:
I don't think that makes sense, this isn't configuring the oracle, but the rpc server